app/views/manifests/show.html.erb

```diff
@@ -57,13 +56,13 @@
       <dt>U.S. EPA Federal Waste Codes</dt>
       <dd>
         <% m["epa_waste_codes"].each do |wc| %>
-          wc
+          wc<% if wc != m["epa_waste_codes"].last %>, <% end %>
```

Comment:
Sounds good, I'll take a stab at it

On Fri, Jan 15, 2016 at 5:16 PM, Jessie A. Young <notifications@github.com>
wrote:

> In app/views/manifests/show.html.erb
> <https://github.com/18F/e-manifest/pull/64#discussion_r49911797>:
>
> > @@ -57,13 +56,13 @@
> >        <dt>U.S. EPA Federal Waste Codes</dt>
> >        <dd>
> >          <% m["epa_waste_codes"].each do |wc| %>
> > -          wc
> > +          wc<% if wc != m["epa_waste_codes"].last %>, <% end %>
>
> I'd prefer to have this logic in a helper method rather than having it in
> the view. Would you be cool w adding that or would you prefer than one of
> us does it?
>
> http://mixandgo.com/blog/the-beginner-s-guide-to-rails-helpers
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/e-manifest/pull/64/files#r49911797>.
>